controllers/submariner/route_agent_resources.go

```diff
@@ -128,4 +134,22 @@ func newRouteAgentDaemonSet(cr *v1alpha1.Submariner, name string) *appsv1.Daemon
 			},
 		},
 	}
+
+	if clusterNetwork.PluginSettings != nil {
```

Comment:
Minor thing but the nil check is unnecessary as Go handles nil map when accessing an element via index.

Reply:
Done.